src/bot/services/admin.py

```diff
@@ -0,0 +1,17 @@
+from src.core.db.models import Admin
+from src.core.db.repository.admin import AdminRepository
+
+
+class AdminService:
+    def __init__(self, admin_repository: AdminRepository, admin_username: str):
+        self._admin_repository = admin_repository
+        self._admin_username = admin_username
+
+    async def add_admin_if_in_settings(self, username: str, instance: Admin):
+        if username == self._admin_username:
+            await self._admin_repository.create(instance)
```

Comment:
Не совсем то что я имел в виду: 
```suggestion
    async def add_admin_if_in_settings(self, instance: Admin):
        if instance.username == self._admin_username:
            await self._admin_repository.create(instance)
```